src/types/contract.rs

```diff
@@ -1,42 +1,160 @@
+pub use build_info::BuildInfo;
 use serde::{Deserialize, Serialize};
 
 /// The struct provides information about deployed contract's source code and supported standards.
 ///
-/// Contract source metadata follows [NEP-330 standard](https://github.com/near/NEPs/blob/master/neps/nep-0330.md) for smart contract verification
+/// Contract source metadata follows [**NEP-330 standard**](https://github.com/near/NEPs/blob/master/neps/nep-0330.md) for smart contracts
+///
+/// See documentation of [`crate::Contract::contract_source_metadata`] on how to query this for a contract via this crate
+// `rustdoc` clearly lacks functionality of automatic backlinks within a single crate
```

Comment:
What do you mean?

Reply:
i meant that the type is used in `crate::Contract::contract_source_metadata`'s signature . So the backlink to `crate::Contract::contract_source_metadata` could be done automatically by rustdoc. 
It would've been too complex for references across crates, but within a single crate it looks like a problem, that has been solved before